samples/azure-quantum/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Process for creating notebook samples for the Azure Quantum portal experience
+This document focuses on the process for creating samples that will eventually appear in the hosted notebooks sample gallery in the portal. If you are creating a sample for other purposes or means of consumption, then this document might not apply to your scenario. (If you are interested in contributing an optimization notebook sample, please see [this repository](https://github.com/microsoft/qio-samples)).
+
+There are three steps involved in creating a new sample.
+
+1. Write the notebook sample
+1. Test it in the portal
+1. Merge it into `main` AND into `feature/samples-gallery`
+
+Once it is merged, a service engineer must be notified to add it to the sample gallery. Once done, the sample will be publicly available in the next deployment, usually within a week.
+
+## Writing the sample
+There are a few principles to keep in mind when writing samples that are going to be presented in the portal's sample gallery.
+1. The command to log in to Azure Quantum should be in its own cell.
+1. There should be no empty cells in the notebook.
```

Comment:
I take it this refers mostly to the last cell that sometimes gets added to notebooks? If so, we may want to run a set of notebook lints or something as part of CI?

Reply:
Indeed, and that could be a good mitigation. For context, empty cells break our E2E tests and detract a bit from the polish.

Reply:
Agreed about the polish, but a bit confused why that breaks e2e; would love to learn more there!